README.md

```diff
@@ -35,6 +35,7 @@ Request bodies should look like
 ```json
 {
   "url": "string",
+  "cloudPlatform": ["gs", "azure","s3", null],
```

Comment:
@nmalfroy -- can we slightly expand the README to indicate that this field is optional and what the fallback behavior is, similar to what we do for `fields` below?